test/mocha/campaigns/PR/9206.js

```diff
@@ -1,35 +1,46 @@
-const {Menu} = require('../../selectors/BO/menu');
-const {SqlManagerPage} = require('../../selectors/BO/advancedParameters/databasePage');
+const {Dashboard} = require('../../selectors/BO/dashboardPage');
+const sqlQueryScenario = require('../common_scenarios/advancedParameters/database');
 
 /** This scenario is based on the bug described in this PR
  * https://github.com/PrestaShop/PrestaShop/pull/9206
  */
 
+let sqlQueryData = [
+  {
+    name: 'shop' + dateTime,
+    sql: 'select * from ps_shop;'
+  },
+  {
+    name: 'address' + dateTime,
+    sql: 'select * from ps_address;'
+  },
+  {
+    name: 'advice' + dateTime,
+    sql: 'select * from ps_advice;'
+  }];
+
 scenario('PR-9206: Check the SQL manager page', () => {
   authentication.signInBO('9206');
-  scenario('Add new SQL query', client => {
-    test('should go to "SQL Manager" page', async () => {
-      await client.waitForAndClick(Menu.Configure.AdvancedParameters.advanced_parameters_menu);
-      await client.waitForAndClick(Menu.Configure.AdvancedParameters.database_submenu, 3000);
+  scenario('Close the onboarding modal if exist ', client => {
+    test('should close the onboarding modal if exist', async () => {
+      await client.waitFor(15000, {waitUntil: 'domcontentloaded'});
+      await client.isVisible(Dashboard.welcome_modal, 3000, {timeout: 90000, visible: true});
+      if (visible) {
+        await client.waitForAndClick(Dashboard.welcome_modal);
+      }
     });
-    test('should click on "Add new SQL query" button', () => client.waitForAndClick(SqlManagerPage.add_new_sql_query_button, 2000));
-    test('should set the "SQL query name" input', () => client.waitForAndType(SqlManagerPage.sql_query_name_input, 'ps_shop' + dateTime, 2000));
-    test('should set the "SQL query" textarea', () => client.waitForAndType(SqlManagerPage.sql_query_textarea, 'select * from ps_shop', 2000));
-    test('should click on "Save" button', () => client.waitForAndClick(SqlManagerPage.save_button, 2000));
-  }, 'common_client');
-  scenario('Search then check that the SQL query was well created', client => {
-    test('should search for the created "SQL query"', () => client.searchByValue(SqlManagerPage.filter_sql_query_name_input, SqlManagerPage.filter_search_button, 'ps_shop' + dateTime));
-    test('should check the appearance of created "SQL query" in the table', () => client.checkTextValue(SqlManagerPage.sql_query_name.replace('%S', 1), 'ps_shop' + dateTime, 'equal', 2000));
-  }, 'common_client');
-  scenario('Delete the created SQL query', client => {
-    test('should click on "Dropdown" button then click on "Delete" action', async () => {
-      await client.waitForAndClick(SqlManagerPage.sql_query_dropdown_button.replace('%S', 1));
-      await client.confirmationDialog();
-      await client.waitForAndClick(SqlManagerPage.sql_query_dropdown_action.replace('%S', 1).replace('%ACTION', 'delete'), 2000);
+    test('should close the "Onboarding" if exists', async () => {
```

Comment:
you have closed the onboarding modal twice. Is there a reason ?

Reply:
I have added a test to close the onBoarding modal and the other one to stop the onBoarding.